pyproject.toml

```
@@ -1,3 +1,67 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "nwbinspector"
+version = "0.6.0"
+description = "Tool to inspect NWB files for best practices compliance."
+readme = "README.md"
+authors = [
+    {name = "Cody Baker"},
+    {name = "Steph Prince"},
+    {name = "Szonja Weigl"},
+    {name = "Heberto Mayorquin"},
+    {name = "Paul Adkisson"},
+    {name = "Luiz Tauffer"},
+    {name = "Ben Dichter",  email = "ben.dichter@catalystneuro.com"}
+]
+urls = { "Homepage" = "https://github.com/NeurodataWithoutBorders/nwbinspector" }
+license = {file = "license.txt"}
+keywords = ["nwb"]
+classifiers = [
+    "Intended Audience :: Science/Research",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Operating System :: POSIX :: Linux",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: MacOS",
+    "License :: OSI Approved :: BSD License",
+]
+requires-python = ">=3.9"
+dependencies = [
+    "pynwb>=2.8",   # NWB Inspector should always be used with most recent minor versions of PyNWB
+    "hdmf-zarr",
+    "fsspec",
+    "s3fs",
+    "requests",
+    "aiohttp",
+    "PyYAML",
+    "jsonschema",
+    "packaging",
+    "natsort",
+    "click",
+    "tqdm",
+    "isodate",
+    "numpy>=1.22.0"
```

```suggestion
    "numpy>=1.22.0,<2.0.0" # TODO: add compatibility for 2.0
```

Ahh looks like we still have some 2.0 incompatibilities local to Inspector here: https://github.com/NeurodataWithoutBorders/nwbinspector/actions/runs/10907985074/job/30272984050?pr=507#step:8:414

Will deal with this in a follow-up